cfpb/consumerfinance.gov
# Adds enhancement to wrap long links

This is useful when link text should be exact urls

Testing:
 - Make sure you have a recent db dump
 - Pull and build
 - Visit http://localhost:8000/admin/pages/18325/edit/preview/
 - Note that long links in the references section are wrapping [eg, 61](http://localhost:8000/admin/pages/18325/edit/preview/?mode=#61)

<img width="767" alt="Screenshot 2024-03-20 at 10 24 47 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/1558033/29e836e5-2e21-4836-a1e9-a5c697c67042">
